Capstone/Detail View/DetailsViewController.h

```diff
@@ -7,7 +7,9 @@
 #import "UIKit/UIKit.h"
 #import "College.h"
 NS_ASSUME_NONNULL_BEGIN
-@interface DetailsViewController : UIViewController
+@interface DetailsViewController : UIViewController<CLLocationManagerDelegate, MKMapViewDelegate> {
```

Comment:
are there methods from `CLLocationManagerDelegate` & `MKMapViewDelegate` that we need to implement? i see that we set the delegate of `locationManager` to `self`, which usually means that there's some functionality in `self` that `locationManager` delegates to

where is `MKMapViewDelegate` used?

Reply:
I'm using the delegate method of location manager to request and get the user's current location using the delegate methods startUpdatingLocation and requestWhenInUseAuthorization. I accidentally put the MKMapViewDelegate there by accident and overlooked it. It should have not been there.